sphinxlint/rst.py

```diff
@@ -4,6 +4,7 @@
 - All constants are ALL_CAPS
 - All compiled regexes are suffixed by _RE
 """
+
```

Comment:
Black also does this with the latest version of black; we just had an old version of black pinned in CI